reaper/cogs/log_reading.py

```diff
@@ -243,6 +244,20 @@ async def on_message(self, message):
                 problem_found = True
                 framework_error = True
 
+            elif "SCRIPT ERROR" in line:
+                dm_log.add_field(
+                    name="",
+                    value="SCRIPT ERROR: True",
+                    inline=False,
+                )
+                logger.info("I found a script error!")
+                problem_found = True
+                j = i
+                while lines[j] != "" and j < i + 15:
```

Comment:
Using a named variable instead of a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) for the `15` would be preferred ^^"